File: Lib/Telerik/Examples/QuickStart/ChartView/ChartTypes/Waterfall/DataModel.vb
Namespace Telerik.Examples.WinControls.ChartView.ChartTypes.Waterfall
	Public NotInheritable Class DataModel
		Private Sub New()
		End Sub
		Public Shared Function GetData() As DataTable
			Dim data As New DataTable("Europe")

			data.Columns.Add("Value", GetType(Decimal))
			data.Columns.Add("Category", GetType(String))
			data.Columns.Add("Summary", GetType(Boolean))
			data.Columns.Add("Total", GetType(Boolean))

			data.Rows.Add(40000, "Beginning" & vbLf & "Balance", False, False)
			data.Rows.Add(15000, "Jan", False, False)
			data.Rows.Add(14000, "Feb", False, False)
			data.Rows.Add(-5000, "Mar", False, False)
			data.Rows.Add(64000, "Q1", True, False)
			data.Rows.Add(-20000, "Apr", False, False)
			data.Rows.Add(-8000, "May", False, False)
			data.Rows.Add(10000, "Jun", False, False)
			data.Rows.Add(-18000, "Q2", True, False)
			data.Rows.Add(46000, "Ending" & vbLf & "Balance", False, True)

			Return data
		End Function
	End Class
End Namespace

File: Lib/Telerik/Examples/QuickStart/ChartView/ChartTypes/Waterfall/DataModel.cs
using System.Data;

namespace Telerik.Examples.WinControls.ChartView.ChartTypes.Waterfall
{
    public static class DataModel
    {
        public static DataTable GetData()
        {
            DataTable data = new DataTable("Europe");

            data.Columns.Add("Value", typeof(decimal));
            data.Columns.Add("Category", typeof(string));
            data.Columns.Add("Summary", typeof(bool));
            data.Columns.Add("Total", typeof(bool));

            data.Rows.Add(40000, "Beginning\nBalance", false, false);
            data.Rows.Add(15000, "Jan", false, false);
            data.Rows.Add(14000, "Feb", false, false);
            data.Rows.Add(-5000, "Mar", false, false);
            data.Rows.Add(64000, "Q1", true, false);
            data.Rows.Add(-20000, "Apr", false, false);
            data.Rows.Add(-8000, "May", false, false);
            data.Rows.Add(10000, "Jun", false, false);
            data.Rows.Add(-18000, "Q2", true, false);
            data.Rows.Add(46000, "Ending\nBalance", false, true);

            return data;
        }
    }
}
